core/primitives-core/src/version.rs

```diff
@@ -422,7 +425,7 @@ pub const MIN_SUPPORTED_PROTOCOL_VERSION: ProtocolVersion = 75;
 const STABLE_PROTOCOL_VERSION: ProtocolVersion = 78;
 
 // On nightly, pick big enough version to support all features.
-const NIGHTLY_PROTOCOL_VERSION: ProtocolVersion = 149;
+const NIGHTLY_PROTOCOL_VERSION: ProtocolVersion = 150;
```

Comment:
I'd suggest going straight for stable - unless this will be a multi PR effort.

Reply:
See comment https://github.com/near/nearcore/pull/13401#discussion_r2067177472

Reply:
Ignore comment above, moved feature straight to stable.